Userland/Libraries/LibJS/Runtime/MathObject.cpp

```diff
@@ -428,6 +429,17 @@ JS_DEFINE_NATIVE_FUNCTION(MathObject::hypot)
     return Value(::sqrt(hypot.as_double()));
 }
 
+JS_DEFINE_NATIVE_FUNCTION(MathObject::imul)
+{
+    auto a = vm.argument(0).to_u32(global_object);
+    if (vm.exception())
+        return {};
+    auto b = vm.argument(1).to_u32(global_object);
+    if (vm.exception())
+        return {};
+    return Value(static_cast<i32>(a * b));
```

Comment:
This looks like it would fail if either of the inputs were a negative number.

Reply:
I'm just following the specification: https://tc39.es/ecma262/#sec-math.imul

Reply:
Oh, okay! :)